jobserver/templates/partials/header.html

```diff
@@ -17,7 +17,7 @@
       </ul>
 
       {% if not user.is_authenticated %}
-        <a class="btn btn-outline-light mt-2 mb-3 my-lg-0" href="{% url "social:begin" "github" %}?next={{ request.path }}" rel="nofollow">Login</a>
+        <a class="btn btn-outline-light mt-2 mb-3 my-lg-0" href="{% url "login" %}" rel="nofollow">Login</a>
```

Comment:
I think we still want `?next={{ request.path }}` here to persist the current URL.

Reply:
Yup, agreed! Thanks for spotting this.